### ISSUE TITLE
Jekyll Site Migration

### ISSUE THREAD

**Issue by Bobfrat** [ioos/ioosngdac#147]
GOAL: GitHub Jekyll site is the primary (only) Github interface for the glider DAC, is up to date and provides users with the most current information.

- [x] Migrate existing wiki to the Jekyll site. 
- [ ] Review and update information.
- [ ] Sunset old wiki site.

**Comment by Bobfrat** [comment]
@mwengren do you have any other info to add to this task?

**Comment by Bobfrat** [comment]
As far as I can tell, the first item has been completed and is available here:
https://ioos.github.io/ioosngdac/

The README points to this site, as does the link on gliders.ioos.us.

Next step is to review and update all the info on the wiki. 

@kerfoot maybe we can tag team this part?

**Comment by mwengren** [comment]
Looks good to me.  Content and link updates are probably the focus for what still needs to be done.  

The main fix that I did for the link above was to update our documentation Jekyll template for only the ioosngdac site so the left nav bar doesn't get cut off in some circumstances.

**Comment by kerfoot** [comment]
Will update the NetCDF specification on the wiki for this repo, then migrate to Jekyll.

**Comment by mwengren** [comment]
@kerfoot We're making some updates to the Jekyll site framework at the moment, so I reviewed the NGDAC site quickly.  I understand you're updating the file format spec in the wiki first with new changes per above comment that will then migrate to Jekyll, which is fine.  

Related question about the Jekyll site however: at one point we modified the sidebar in the Jekyll template for the ioos/ioosngdac repo specifically so that it wouldn't use CSS 'fixed' positioning (this was because the menu at one point had too many elements to fit on a user's screen). 

It looks like all those menu elements have been [removed](https://github.com/ioos/ioosngdac/blob/gh-pages/_data/sidebars/sidebar_ioos.yml) since then?  I think they were links to each variable in the File Format spec.  Should we update the master Jekyll template so the sidebar for the NGDAC site works like the others (floating/always visible)?
